File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldUserId.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnRowProperty;
import com.neome.api.meta.base.Types.EnumDefnUserProperty;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldUserId extends DefnFieldEditable
{
  @Nullable
  public EnumDefnRowProperty defaultValue;

  @Nullable
  public EnumDefnUserProperty displayProperty;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldUserId.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.EnumDefnRowProperty
import com.neome.api.meta.base.Types.EnumDefnUserProperty

interface DefnFieldUserId : DefnFieldEditable
{
  val defaultValue: EnumDefnRowProperty?
  val displayProperty: EnumDefnUserProperty?
}